### ISSUE TITLE
[NEWS] Plan wygaszenia WebAPI / WebAPI phase out

### ISSUE THREAD

**Issue by PawelTaberski** [allegro/allegro-api#3492]
**Wyłączymy WebAPI:**

- **1.09.2020** na Sandboxie. 
- **26.10.2020** na środowisku produkcyjnym.

Terminy zaktualizowaliśmy, ponieważ od października w WebAPI nie pozostaną już żadne metody, które uzasadniałyby jego dłuższe utrzymywanie.

**26.10.2020** wyłączymy także metody do:

- zarządzania wypłatami - [doRequestPayout](https://allegro.pl/webapi/documentation.php/show/id,729),
- pobierania informacji dot. ustawień Allegro Finanse - [doGetMyPaymentsInfo](https://allegro.pl/webapi/documentation.php/show/id,725).

Z powodu niskiego użycia, nie będziemy dodawać ich odpowiedników w REST API. Aktualne ustawienia danych do wypłat możesz sprawdzić w [Moim Allegro](https://allegro.pl/myaccount/Payu/PayoutsConfiguration.php). Jeśli chcesz otrzymywać codziennie wypłaty, wystarczy, że ustawisz automatyczny sposób wypłat.

---

**We will disable Allegro WebAPI from:**

- **1.09.2020** on Sandbox. 
- **26.10.2020** on production environment.

We have updated the terms because from October, there will be no methods left in WebAPI that justify longer maintenance.

**On October 26th, 2020** we will also disable WebAPI methods for:

- managing payouts - [doRequestPayout](https://allegro.pl/webapi/documentation.php/show/id,1149),
- retrieving Allegro Finance settings of a logged-in user. - [doGetMyPaymentsInfo](https://allegro.pl/webapi/documentation.php/show/id,1143).

Due to low usage, we do not plan to release the equivalent in REST API. You can check the current data settings for withdrawals in [My Allegro](https://allegro.pl/myaccount/Payu/PayoutsConfiguration.php). If you want to receive daily payouts, all you need to do is select up an automatic payout option.

**Comment by FromAnyHole** [comment]
To zróbcie to jak LUDZIE dla LUDZI, no i niech w tej dacie wygasa już WSZYSTKO - bez wcześniejszego wyłączania metod przeniesionych na 30.09 (z końca tego roku). Ale po co my to piszemy...

**Comment by manoos** [comment]
A jest dostępny odpowiednik metody: [doShowItemInfoExt](https://allegro.pl/webapi/documentation.php/show/id,342#method-input) ?

**Comment by MaciejFrackowiak** [comment]
@manoos Nie, odpowiednika w REST API nie będzie - informowaliśmy o tym w poście #3377
